src/services/elasticsearch.js

```diff
@@ -31,6 +32,27 @@ class ElasticSearchService {
     }
     return [];
   }
+
+  static async addQuote({ playName, lineNumber, speaker, quoteText }) {
+    // Construct the elastic search specific data model
+    const q = {
+      index: 'quotes',
+      type: 'line',
+      id: shortid.generate(),
+      body: {
+        play_name: playName,
+        line_number: lineNumber,
+        speaker,
```

Comment:
I think this might be missing a key, or maybe this is some kind of destructuring that I'm not familiar with?